crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -2150,8 +2160,11 @@ fn test_extra_arg(
 
     let (a, conds) = cond_test_sig(
         &format!(
-            "((({{h1}} ({{h2}} (123 ((({} ({} ( 1337 ) {} ))))",
-            condition as u8, arg, extra_cond
```

Comment:
this test isn't really about testing DEDUP. I think it would be fine to just change the expectation of the `ELIGIBLE_FOR_DEDUP` flag being set.